src/parser.rs

```diff
@@ -73,8 +93,18 @@ pub fn parse_and_generate_code(schema_file_contents: String, config: &Generation
         }
     }
 
+    let mut tables_hm = std::collections::HashMap::new();
+
     for table in tables.iter_mut() {
-        table.generated_code = code::generate_table(table.clone(), config);
+        // table.generated_code = code::generate_table(table.clone(), config);
+        tables_hm.insert(table.name.clone(), table);
```

Comment:
this looks like a bit more than just a fmt change, will this modify the behavior of the program?